source/_guides/appstore-publishing/publishing-an-app.md

```diff
@@ -50,103 +50,69 @@ resources are before creating the listing.
-   it provides:
-
-    ![app-category](/images/guides/appstore-publishing/app-category.png)
-
-4. Upload the large and small icons representing the app:
-
-    ![app-icons](/images/guides/appstore-publishing/app-icons.png)
+3. Click 'Next' to be taken to the images page. Here you should upload screenshots for each platform your App supports.
+    ![add-images](/images/guides/appstore-publishing/new-add-image.png)
 
-5. Click 'Create' to be taken to the listing page. This page details the status
-   of the listing, including links to subpages, a preview of the public page,
-   and any missing information preventing release.
+   Further down the page is the section to upload a banner. During inital submission, you can provide one banner. After
+   your app is submitted, you will be able to provide up to 3 banners per platform if desired.
 
-    ![app-listing](/images/guides/appstore-publishing/app-listing.png)
+   You must also provide a large icon and small icon for your Watch App.
 
-6. The status now says 'Missing: At least one published release'. Click 'Add a
-   release' to upload the `.pbw`, optionally adding release notes:
+   ![add-banners](/images/guides/appstore-publishing/new-add-banner.png)
 
-    ![app-release](/images/guides/appstore-publishing/app-release.png)
+4. The next step is to optionally provide a website link and a source code link. Both fields can be left blank if desired.
 
-7. Click 'Save'. After reloading the page, make the release public by clicking
-   'Publish' next to the release:
+    ![face-release](/images/guides/appstore-publishing/new-add-source.png)
 
-    ![face-release-publish](/images/guides/appstore-publishing/face-release-publish.png)
+5. Click 'Next'. The final page requires you to provide release notes for your first release, as well as the .pbw file.
 
-8. The status now says 'Missing: A complete X asset collection' for
-   each X supported platform. Click 'Manage Asset Collections', then click
-   'Create' for a supported platform.
+    ![face-release-publish](/images/guides/appstore-publishing/new-add-release.png)
 
-9. Add a description, up to 5 screenshots, optionally up to three header images,
-   and a marketing banner before clicking 'Create Asset Collection'.
+   Once you have submitted this information, and you are happy with the submission, press 'Publish App to Store' to submit your app!
 
-    ![app-assets](/images/guides/appstore-publishing/app-assets.png)
 
-10. Once all asset collections required have been created, click 'Publish' or
-    'Publish Privately' to make the app available only to those viewing it
-    through the direct link.
-
-11. After publishing, reload the page to get the public appstore link for social
-    sharing, as well as a deep link that can be used to directly open the
-    appstore in the mobile app.
+## Publishing a Companion App
 
+<div class="alert alert--fg-white alert--bg-orange">
```

Comment:
We should consider different colours here, since the current combo doesn't meet accessibility requirements:

<img width="677" height="172" alt="image" src="https://github.com/user-attachments/assets/06f829c3-e600-4fea-83d4-4d9a9cee93f5" />